tests/particle_attributes/particle_attributes.xml

```diff
@@ -5,94 +5,118 @@
   <owner userid="cmath"/>
   <tags>flml</tags>
   <problem_definition length="short" nprocs="1">
-    <command_line>        
+    <command_line>
 ../../bin/fluidity -v3 -l particle-attributes.flml
     </command_line>
   </problem_definition>
   <variables>
     <variable name="attributes_error_1" language="python">
 import numpy as np
-from fluidity_tools import stat_parser
-s = stat_parser("particle_attributes.particles.Thomas")
-attributes_error = np.zeros((1,60))
+import h5py
 
-for i in range(60):
-    name = 'Thomas_' + str(i+1).zfill(2)
-    attributes_error[0,i] = s[name]['Chem1'][0][-1] - 5.0
+f = h5py.File('particle_attributes.particles.Thomas.h5part', 'r')
+n = len(f)-1
+att_chem1 = f[f'/Step#{n}/Chem1'][:]
```

Comment:
Format strings are python3. As I said, I'd be for moving to python3 exclusively, but that would need to happen first then...